abce/__init__.py

```diff
@@ -391,32 +393,36 @@ def build_agents(self, AgentClass, group_name, number=None,
             number:
                 number of agents to be created.
 
-            parameters:
-                a dictionary of parameters
-
             agent_parameters:
                 a list of dictionaries, where each agent gets one dictionary.
                 The number of agents is the length of the list
 
+            any other parameters:
+                are directly passed to the agent
+
         Example::
 
          firms = simulation.build_agents(Firm, 'firm',
              number=simulation_parameters['num_firms'])
          banks = simulation.build_agents(Bank, 'bank',
-                                         parameters=simulation_parameters,
                                          agent_parameters=[{'name': 'UBS'},
-                                         {'name': 'amex'},{'name': 'chase'})
+                                         {'name': 'amex'},{'name': 'chase'}
+                                         **simulation_parameters,
+                                         loanable=True)
 
          centralbanks = simulation.build_agents(CentralBank, 'centralbank',
                                                 number=1,
-                                                parameters={'rounds':
-                                                             num_rounds})
+                                                rounds=num_rounds)
         """
         assert number is None or agent_parameters is None, \
             'either set number or agent_parameters in build_agents'
+        if agent_parameters is None:
+            agent_parameters = {}
+        if parameters is None:
```

Comment:
This check isn't necessary, since there is no longer `parameters`.

Reply:
Ah nvm, I looked at the init of the docstring section, instead of the actual def line.